packages/lib/src/settings/options/itemEnableShadow.js

```diff
@@ -3,6 +3,8 @@ import { default as GALLERY_CONSTS } from '../../common/constants';
 
 export default {
   title: 'Enable Item Shadow',
+  isRelevantDescription:
+    'Set "Crop Images" to "false" or set "Crop Type" to anything but "Fit" and set "Choose info layout" to "Attached Background" or set "Texts Placement" to "Show On Hover"',
```

Comment:
First Set "Crop Images" to "false" or set "Crop Type" to anything but "Fit".
Then set "Choose info layout" to "Attached Background" or set "Texts Placement" to "Show On Hover".